docs/development/style-guide.rst

```diff
@@ -0,0 +1,237 @@
+********************************************************
+Astropy Style Guide: A Writing Resource for Contributors
+********************************************************
+
+Abbreviations
+=============
+
+Place abbreviations such as i.e. and e.g. within parentheses, where they are
+followed by a comma. Alternatively, consider using "that is" and “for example”
+instead, preceded by an em dash or semicolon and followed by a comma, or
+contained within em dashes.
+
+Examples:
```

Comment:
Following numpydoc formatting, there is no colon after "Examples". You will also need to readjust the following dashed line length accordingly. This comment applies to all following occurrences.
```suggestion
Examples
```

Reply:
Got it, thanks!